README.md

```diff
@@ -1,3 +1,295 @@
+
+### landscape.cluster
+```yaml
+cluster:                                            # Information about your base cluster
+  kubeconfig: <relative path + filename>            # Path to your `kubeconfig` file, relative to folder `landscape`   
+  domain: <prefix>.<cluster domain>                 # Unique basis domain for DNS entries
+  iaas: <gcp|aws|azure>                             # IaaS provider (coming soon: openstack|alicloud)
+  region: <major region>-<minor region>             # Example (gcp, aws): europe-west1; example (Azure): westeurope
+```
+
+Information about your base cluster, where the Gardener will be deployed on.
+
+| Field | Type | Description | Example |
+|:------|:--------|:--------|:--------|
+|`kubeconfig`|File path| Path to your kubeconfig, relative to your landscape folder. It is recommended to create a kubeconfig file in your landscape folder to be able to sync all files relevant for your installation with a git repository.| `./kubeconfig` |
+|`domain`| Unique name| Basis domain for DNS entries. As a best practice, use an individual prefix together with the cluster domain of your base cluster.|`vedge.gcp.dev.k8s.jacksgrocerystore.com`|
+|`iaas`| Fixed value | IaaS provider you would like to install Gardener on. | `gcp` |
+|`region`|IaaS provider specific| Region where your Kubernetes base cluster is deployed. | `europe-west1` (GCP)|
+
+### landscape.networks
+```yaml
+networks:                                 # CIDR IP ranges of base cluster
+  nodes: <CIDR IP range>                                  
+  pods: <CIDR IP range>                                   
+  services: <CIDR IP range>
+```
+CIDR ranges of your base cluster.
+
+Finding out CIDR ranges of your cluster is not trivial. For example, GKE only tells you a "pod address range" which is actually a combination of pod and service CIDR. However, since the `kubernetes` service typically has the first IP of the service IP range and most methods to get a kubernetes cluster tell you at least something about the CIDRs, it is usually possible to find out the CIDRs with a little bit of educated guessing.
+
+### landscape.iaas
+```yaml
+iaas:
+  region: <major region>-<minor region>             # region that Gardener will use for seed clusters
+  zones:                                            # Remove zones block for providers other than GCP or AWS
+    - <major region>-<minor region>-<zone>          # Example: europe-west1-b
+    - <major region>-<minor region>-<zone>          # Example: europe-west1-c     
+    - <major region>-<minor region>-<zone>          # Example: europe-west1-d
+  credentials:                                      # credentials to get access to the seed cluster through service account
+```
+Contains the information where Gardener will create seed clusters. By default, the *initial* seed component will create a seed resource using your base cluster as seed cluster.
```

Comment:
The use of *seed* clusters here (and some lines above and below) is wrong. It is used to configure an initial seed and default  profile used to create *shoot* clusters. 

Other seeds and profiles can be added later on.

Reply:
@mandelsoft: 
I hopefully found all cases and explain it now in the way you described it above.

Could you give me an idea what "profiles" are? Is that something users will be able to see and edit on the Gardener dashboard after the installation?

Reply:
"profile" means a `Cloudprofile" (e.g. https://github.com/gardener/gardener/blob/master/example/30-cloudprofile-gcp.yaml). Cloudprofiles are not directly visible in the Gardener Dashboard, but they define the constraints for shoots. For example, you can only choose a specific kubernetes version for your shoot, if that version is contained in a Cloudprofile. To link it to the Dashboard, the Cloudprofiles define which values can be chosen in the drop-down menus.